packages/playground/src/weblets/tf_algorand.vue

```diff
@@ -152,9 +152,10 @@
       <input-tooltip inline tooltip="Renting capacity on certified nodes is charged 25% extra.">
         <v-switch color="primary" inset label="Certified" v-model="certified" :disabled="loadingFarm" hide-details />
       </input-tooltip>
-
+      <NodeSelector v-model="selection" />
       <SelectFarmManager>
         <SelectFarm
+          v-if="selection == 'automated'"
```

Comment:
as we have the `Selection` enum, why we still use the `hard-coded`
```suggestion
          v-if="selection == "Selection.AUTOMATED"
```